File: koku-frontend/src/app/formular/form-outlet.ts
export const FORM_OUTLET = {
  CONTENT: 'content',
  PREPEND_OUTER: 'prependOuter',
  PREPEND_INNER: 'prependInner',
  APPEND_INNER: 'appendInner',
  APPEND_OUTER: 'appendOuter',
} as const;

export type FormOutlet = (typeof FORM_OUTLET)[keyof typeof FORM_OUTLET];

File: formular/src/main/java/de/domschmidt/formular/factory/FormOutlet.java
package de.domschmidt.formular.factory;

public enum FormOutlet {
    CONTENT("content"),
    PREPEND_OUTER("prependOuter"),
    PREPEND_INNER("prependInner"),
    APPEND_INNER("appendInner"),
    APPEND_OUTER("appendOuter");

    private final String name;

    FormOutlet(final String name) {
        this.name = name;
    }

    public String outletName() {
        return this.name;
    }
}
